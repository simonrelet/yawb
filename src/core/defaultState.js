export default Object.freeze({
  hasNewVersion: false,
  theme: 'default',
});
