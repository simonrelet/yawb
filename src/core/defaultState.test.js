import defaultState from './defaultState';

describe('defaultState', () => {
  it('should have a state', () => {
    expect(defaultState).toMatchSnapshot();
  });
});
