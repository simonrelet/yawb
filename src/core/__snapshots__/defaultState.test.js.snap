// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`defaultState should have a state 1`] = `
Object {
  "hasNewVersion": false,
  "theme": "default",
}
`;
